policy/modules/contrib/insights_core.te

```diff
@@ -48,6 +48,7 @@ manage_dirs_pattern(insights_core_t, insights_core_tmp_t, insights_core_tmp_t)
 manage_files_pattern(insights_core_t, insights_core_tmp_t, insights_core_tmp_t)
 manage_chr_files_pattern(insights_core_t, insights_core_tmp_t, insights_core_tmp_t)
 manage_lnk_files_pattern(insights_core_t, insights_core_tmp_t, insights_core_tmp_t)
+files_rw_var_lib_dirs(insights_core_t)
```

Comment:
Can you provide more details on what flatpak is doing? What is the path it is writing to?

Reply:
The core just needs to execute the `flatpak list` command ([here](https://github.com/RedHatInsights/insights-core/blob/master/insights/specs/default.py#L319)), I guess, it's because that the `flatpak list` command needs to create/access some `cache` like temp dirs?

Maybe this https://github.com/flatpak/flatpak/issues/4128 would fix this?

Reply:
Correct, the type for /var/lib/flatpak should be different, but that's another issue.